src/convert.jl

```diff
@@ -986,3 +986,43 @@ function convert(::Type{Zonotope},
     Z2 = translate(Z1, vector(am), share=true)
     return Z2
 end
+
+"""
+    convert(::Type{<:HParallelotope}, Z::AbstractZonotope{N}) where {N}
+
+Converts a zonotopic set of order one into a parallelotope in constraint
+representation.
+
+### Input
+
+- `HParallelotope` -- type used for dispatch
+- `Z`              -- zonotopic set
+
+### Output
+
+A parallelotope in constraint representation.
+
+### Notes
+
+This function requires that the list of constraints of `Z` are obtained in
+the particular order returned from the constraints list function of a `Zonotope`.
```

Comment:
```suggestion
```
This note does not match the implementation: you convert to `Zonotope`, so this is guaranteed.

Reply:
yes, i'd like to make emphasis precisely on that conversion. if you want to phrase it in other way go ahead